packages/integration-karma/test/component/LightningElement.dispatchEvent/index.spec.js

```diff
@@ -32,7 +31,7 @@ it('should throw an error if the parameter is not an instance of Event', () => {
     }).toThrowError();
 });
 
-it('should throw when event is dispatched during construction', function () {
+it('should not throw when event is dispatched during construction', function () {
```

Comment:
This is kind of a weird test to have outside the context of this PR, maybe we should just delete it?

Reply:
That what I originally did. However, I also wanted to make sure that dispatching an event in the middle of the constructor doesn't hose the component. I can change the test name if you have a better proposal.